### PR TITLE
Improvements and added Category order section

### PR DESCRIPTION
**Pull request by joseph619**

### CODE REVIEW

**Comment by GHOSTBEAR** [comment]
Will create a video for this. Probably by the end of this weekend

File: src/help/guides/categories.md [hunk 1]
```diff
@@ -49,9 +74,11 @@ You can delete a Category in `Triple dots > Edit Categories` then long press the
 :::
 
 ## Add Manga to Category
-You can add a Manga to a Category by long pressing on the one you want to add. Then press the square looking icon. Thereafter select which category you want it in. You can also add multiple manga to a category by selecting them when you see the square looking icon.
+
+You can add a Manga to a Category by long pressing on the one you want to add. Then press the square looking icon (in the middle between pencil and three dots). Thereafter select which category you want it in. You can also add multiple manga to a category by selecting them when you see the square looking icon.
+
 ::: tip
-You can add a Manga to multiple Categories by selecting the categories you want to have it in
+You can also add a Manga to multiple Categories by selecting the categories you want to have it in
```
Comment (Soitora) [inline src/help/guides/categories.md]: If I recall correctly Manga doesn't need any word indicating singular/plural, it's "also add manga to", not "a manga", "an manga" or "multiple manga", it's just "manga". This occurs at least in one other place (row: 78).

File: src/help/guides/categories.md [hunk 2]
```diff
@@ -34,8 +39,28 @@ You can edit a Category in `Triple dots > Edit Categories` then long press the c
...
+		<source :src="$withBase('/assets/media/reordercategories.webm')" type="video/webm" />
+		<source :src="$withBase('/assets/media/reordercategories.mp4')" type="video/mp4" />
+	</video>
+</figure>
+:::
+
 ## Delete a Gategory
-You can delete a Category in `Triple dots > Edit Categories` then long press the category you want to delete, then press the trash can icon.
+
+You can delete a Category in `My Library` > triple dots (`⋮`) > `Edit categories` then long press the category you want to delete, then press the trash can icon.
```
Comment (Soitora) [inline src/help/guides/categories.md]: `then ... , then` flows wrong, maybe:
> You can delete a Category by going to `My Library` > triple dots (`⋮`) > `Edit categories` and long pressing the category you want to delete then pressing the delete button (`🗑️`).

File: src/help/guides/categories.md [hunk 3]
```diff
@@ -34,8 +39,28 @@ You can edit a Category in `Triple dots > Edit Categories` then long press the c
 </figure>
 :::
 
+## Change Category order
+
+You can change the order of Categories, such as bring a certain Category to the top. This category will appear as the first tab on `My Library`.
```
Comment (Soitora) [inline src/help/guides/categories.md]: `bring`... maybe `bringing`?

**Comment by Soitora** [comment]
> Will create a video for this. Probably by the end of this weekend
_@GHOSTBEAR https://github.com/tachiyomiorg/website/pull/52#issuecomment-535932798_

Would be appreciated, just reference this PR in your PR.
Also for both of you, I recommend signing up for the [Hacktoberfest](https://hacktoberfest.digitalocean.com/).

**Comment by GHOSTBEAR** [comment]
@Soitora the video is already in this PR :wink:

File: src/help/guides/categories.md (hunk 1 above)
Comment (joseph619) [reply src/help/guides/categories.md]: Fixed all as requested

**Comment by Soitora** [comment]
> @Soitora the video is already in this PR 😉

Spooky, didn't notice your commit. :)

**Pull request merged**